code/game/gamemodes/cult/hell_universe.dm

```diff
@@ -98,19 +99,21 @@ In short:
 		A.updateicon()
 
 /datum/universal_state/hell/proc/OverlaySet()
-/*
-	for(var/turf/space/spess in world)
-		spess.overlays += "hell01"
-	*/
+	/*this method is not particularly cheaper than for(var/turf/T in world) and has various downsides
```

Comment:
*I meant nonspace turfs get an underlay while space turfs get an overlay. If you notice pomf fixed at some point space not underlay properly underneath shuttle walls.